tests/CommunityToolkit.Mvvm.UnitTests/Test_AsyncRelayCommand.cs

```diff
@@ -289,11 +291,6 @@ private static async Task Test_AsyncRelayCommand_AllowConcurrentExecutions_TestL
         Assert.IsFalse(command.CanBeCanceled);
         Assert.IsFalse(command.IsCancellationRequested);
 
-        Task newTask = command.ExecuteAsync(null);
```

Comment:
The code was checking that if a command was invoked when `CanExecute` was `false`, it just bypassed execution and returned `Task.CompletedTask`. Since that's no longer the case, then this test case was just invalid now.